client/src/cbltest/api/testserver.py

```diff
@@ -14,6 +15,23 @@
 from cbltest.version import VERSION
 
 
+class ServerVariant(Flag):
+    ANDROID = auto()
+    C = auto()
+    DOTNET = auto()
+    IOS = auto()
+    JAVA = auto()
```

Comment:
FWIW, We should, actually, be calling this "JVM".  There are multiple languages (notably Kotlin) that run on the JVM and Java runs on Android.  The distinction between the two things is that one targets ART (on Android) and the other targets the JVM (desktop or webservices).
Perhaps, because this matches our inaccurate naming of the platforms, it is the right thing to do.  Just FYI.
I am entirely willing to change the identifier in the header, if that will help.

Reply:
I can call it JVM if that will help things.